neural_insights/README.md

```diff
@@ -0,0 +1,110 @@
+<div align="center">
+  
+Intel® Neural Insights

```

Comment:
We could not use `Intel@`, this name is not formal approved.

Reply:
Done